src/dialog.h

```diff
@@ -130,6 +145,9 @@ void dialog_draw(Dialog *d);
 bool dialog_page(Dialog *d)
 	attr_nonnull_all;
 
+void dialog_draw_title(Dialog *d, DialogActor *actor, char *name, char *title);
+	attr_nonnull_all;
```

Comment:
```
../../../src/dialog.h:149:2: warning: empty declaration
  149 |  attr_nonnull_all;
      |  ^~~~~~~~~~~~~~~~
```

The `attr_nonnull_all` attribute is supposed to be attached to the function declaration; the semicolon on the first line should be removed.